apps/web/src/app/archive/page.tsx

```
@@ -26,9 +27,11 @@ async function WishlistPage({ searchParams }: WishlistPageProps) {
   const queryClient = getQueryClient();
 
   if (activeTab === '저장한 위시템') {
-    await queryClient.prefetchQuery({
+    await queryClient.prefetchInfiniteQuery({
       queryKey: ['wishlists'],
-      queryFn: getWishlist,
+      queryFn: ({ pageParam }) => getWishlist(pageParam as string | null),
+      initialPageParam: null as string | null,
+      getNextPageParam: (page: WishlistPageT) => (page.hasNext ? page.nextCursor : undefined),
```

ESLint

[error] Unexpected use of undefined.

(no-undefined)